src/testUtils/java/org/broadinstitute/hellbender/testutils/ArgumentsBuilder.java

```diff
@@ -155,6 +155,10 @@ public ArgumentsBuilder addVCF(final Path vcf) {
         return add(StandardArgumentDefinitions.VARIANT_LONG_NAME, vcf);
     }
 
+    public ArgumentsBuilder addVCF(final boolean vcf) {
+        return add(StandardArgumentDefinitions.REANNOTATE_VCF_LONG_NAME, vcf);
```

Comment:
This is a very tool-specific argument, so we don't want it here in `ArgumentsBuilder` as a built-in method, since `ArgumentsBuilder` is used across the GATK and should reference only common / framework-level arguments. 

Remove this overload, and use the existing `add(final String argumentName, final String argumentValue)` or `add(final String argumentName, final File file)` methods in your test code.